src/appleseed/foundation/math/microfacet.cpp

```diff
@@ -368,38 +368,50 @@ Vector3f BeckmannMDF::sample(
             gamma);
 }
 
+
 // This code comes from OpenShadingLanguage test render.
 Vector2f BeckmannMDF::sample11(
     const float         cos_theta,
     const Vector3f&     s,
     const float         gamma) const
 {
-    const float ct = max(cos_theta, 1.0e-6f);
+    const float threshold = 1e-06f;
```

Comment:
It's a constant so it should be spelled `Threshold`.

Sometimes, it doesn't make sense to declare a constant for the threshold/epsilon value. A good hint is in the name of the constant: does `Threshold` tell anything to the reader as to how this value should be tuned and what effect it will have?

I suspect that here you would be better off with using `1.0e-6f` directly where you need it since it's not really something that needs adjusting and it would be hard to give it a more descriptive name.